scripts/configuration/ac-build-config.xml

```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0"?>
 <config>
     <sdkInfo>
-        <sdkVersion>1.10.1-SNAPSHOT</sdkVersion>
-        <iosVersion>1.10.0</iosVersion>
-        <androidVersion>1.9.0</androidVersion>
+        <sdkVersion>1.11.0-SNAPSHOT</sdkVersion>
+        <iosVersion>1.11.0</iosVersion>
+        <androidVersion>1.10.0</androidVersion>
```

Comment:
This usually requires some quick manual testing in Forms.Puppet applications for iOS and Android before going further into the release process.

Reply:
I will do this testing